perception/lidar_centerpoint/schema/centerpoint.schemal.json

```diff
@@ -0,0 +1,98 @@
+{
```

Comment:
@badai-nguyen This file has a wrong extension. (`.schemal.json`)

Might be intentional too, I don't know.

Maybe related to https://github.com/autowarefoundation/autoware-github-actions/pull/293

Reply:
@xmfcx Thank you for your comment. It was my mistake. I will fix it.